test/sideof.jl

```diff
@@ -15,6 +15,13 @@
   pts = [p1, p2, p3]
   @test sideof(pts, c) == [IN, OUT, ON]
 
+  c_mm = LengthUnit(Unitful.mm)(c)
+  @test sideof(p1, c) == IN
+  @test sideof(p2, c) == OUT
+  @test sideof(p3, c) == ON
+  pts = [p1, p2, p3]
+  @test sideof(pts, c) == [IN, OUT, ON]
```

Comment:
tests are still using the old `c` variable instead of `c_mm`

Reply:
dang, trying to go too fast with this PR. fixed.